pkgs/top-level/all-packages.nix

```diff
@@ -16075,6 +16075,13 @@ with pkgs;
     bintools = bintoolsDualAs;
   });
 
+  gnat12Packages = recurseIntoAttrs (callPackage ./ada-packages.nix { gnat = buildPackages.gnat12; });
+  gnat13Packages = recurseIntoAttrs (callPackage ./ada-packages.nix { gnat = buildPackages.gnat13; });
+  gnatPackages   = recurseIntoAttrs (callPackage ./ada-packages.nix { gnat = buildPackages.gnat; });
```

Comment:
Would still be nice to make this an alias to avoid duplicate eval work. Maybe move it next to the binding for `gnat = `, so you can't forget to update it?

Reply:
Ack, I've added a comment behind `gnat=gnat12` to also update `gnatPackages`.